### PR TITLE
spanish for review

### PR DESCRIPTION
**Pull request by captainbuckkets**

### CODE REVIEW

**Comment by captainbuckkets** [comment]
Check and make sure that the json format is compatible before I implement more of the translations.  Also not sure on how we want to name the folders.

File: public/locales/en-US/translation.json [hunk 1]
```diff
@@ -269,7 +269,7 @@
       "text5": "Run <code>daemon\\peercoind</code>",
       "text6": "Your wallet is located in <code>C:\\Users\\{username}\\AppData</code> (hidden folder).",
       "text7": "Open the folder with <code>Start menu -> Run...</code> (or press <code>Winkey-R</code>), type <code>%APPDATA%\\PPCoin</code> into the field, and press Enter.",
-      "text8": "Additional help can be found in the <a href=\"https://docs.peercoin.net/\">Peercoin Documentation</a> or by asking on our <a href=\"https://talk.peercoin.net/\">Forums</a>."
+      "text8": "Ayuda adicional se puede encontrar de peercoin en su <a href=\"https://docs.peercoin.net/\">Documentación</a> or by asking on our <a href=\"https://talk.peercoin.net/\">Forums</a>."
```
Comment (kazzkiq) [inline public/locales/en-US/translation.json]: This is the english file, texts should remain in english.

**Comment by kazzkiq** [comment]
Everything ok, the file is valid you're good to go.

File: public/locales/en-US/translation.json (hunk 1 above)
Comment (captainbuckkets) [reply public/locales/en-US/translation.json]: Yes, that was a mistake in editing.  Thanks for catching it.

**Comment by captainbuckkets** [comment]
I think I got everything.  There is a lot so its possible I missed something but I double checked it again

**Pull request closed without merging**